django_extensions/management/commands/sync_s3.py

```diff
@@ -66,7 +66,7 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand, CommandError
-from django.utils.six import StringIO
+from django.utils.six import BytesIO
```

Comment:
I know that Django has `six` module for compatibility but now Django 2.0 is Python 3+ only and in future versions they will probably delete this module. Also `io` module is available as of Python 2.6, and `BytesIO` can be imported from there and will work in Python 2 and 3 (and also is written in C and should be faster than SringIO in Python 2).

I suggest that `from io import BytesIO` should be preferred then `from django.utils.six import BytesIO` for Python 2/3 compatible code which should support Python 2.6+